client/src/components/NavBar/AuthBars/Notifications/Notification/Notification.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+
+interface props {
+  description: string;
+  title: string;
+  date: string;
+  image: string;
+}
+
+const Notification = (props: props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.notification}>
+      <Box className={classes.imageContainer}>
+        <img src={props.image} className={classes.image} />
```

Comment:
It's a good practice to add the `alt` attribute to `img` tag as it can be helpful to screen readers.